app.js

```diff
@@ -1,185 +1,249 @@
+  this.randomOutput = function(){
     return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
-  },
-  dailyRandomOutput: function(){
+  };
+  this.dailyRandomOutput = function(){
     var cookiesArray = [];
     for(i = 0; i < 8; i++){
       var arrayValue = this.randomOutput();
       cookiesArray.push(arrayValue);
     }
     return cookiesArray;
-  },
-  randomOutputSum: function(){
-    var arraySum = this.dailyRandomOutput();
-    var sum = 0;
-    for(i = 0; i < arraySum.length; i++){
-      sum += arraySum[i];
-    }
-    arraySum.push(sum);
-    console.log(arraySum);
-    return arraySum;
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('seatac_airport');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
+  };
+  this.randomOutputSum = function(){
+      var arraySum = this.dailyRandomOutput();
+      var sum = 0;
+      for(i = 0; i < arraySum.length; i++){
+        sum += arraySum[i];
+      }
+      arraySum.push(sum);
+      console.log(arraySum);
+      return arraySum;
     };
+  this.storeLabel = function(){
+    var el = document.getElementById('name');
+    var tableRowElement = document.createElement('tr');
+    var updateTableRow = el.appendChild(tableRowElement);
+    updateTableRow.textContent = this.name;
   }
-};
-seaTacAirport.renderManipulation();
-
-var southcenter = {
-  min: 11,
-  max: 38,
-  avg: 1.9,
-  randomOutput: function(){
-    return Math.floor(((Math.random()*(this.max - this.min + 1)) + this.min) * this.avg);
-  },
-  dailyRandomOutput: function(){
-    var cookiesArray = [];
-    for(i = 0; i < 8; i++){
-      var arrayValue = this.randomOutput();
-      cookiesArray.push(arrayValue);
+  this.renderTable = function(){
+    var outputArr = this.randomOutputSum();
+    console.log(outputArr);
+    for(i=0; i<outputArr.length; i++){
+      var el = document.getElementById('index' + i);
+      var tableRowElement = document.createElement('tr');
+      var updateTableRow = el.appendChild(tableRowElement);
+      updateTableRow.textContent = outputArr[i];
     }
-    return cookiesArray;
-  },
-  randomOutputSum: function(){
-    var arraySum = this.dailyRandomOutput();
-    var sum = 0;
-    for(i = 0; i < arraySum.length; i++){
-      sum += arraySum[i];
-    }
-    arraySum.push(sum);
-    console.log(arraySum);
-    return arraySum;
-  },
-  renderManipulation: function(){
-    var arrSum = this.randomOutputSum();
-    var x = document.getElementById('southcenter');
-    for(i=0;i<9;i++){
-      var el = document.createElement('li');
-      var updatedListItem = x.appendChild(el);
-      updatedListItem.textContent = timeLabelsArray[i] + arrSum[i];
-    };
   }
-};
-southcenter.renderManipulation();
+}
+var pikePlace = new Store ("Pike Place", 17, 88, 5.2);
```

Comment:
Nice lines here. Create an instance and then call the two methods needed to create the store and the table row. Very clear and clean.